src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -1123,6 +1123,16 @@ public bool LoadCurrentSegment()
                 return false;
             }
 
+            public bool GetCurrentSegment()
```

Comment:
GetCurrentSegment vs LoadCurrentSegment
Impl is different, but from naming point of view those are doing the same thing. Please name them accordingly and write a comment.